packages/mdc-menu-surface/constants.ts

```diff
@@ -21,7 +21,8 @@
  * THE SOFTWARE.
  */
 
-/** @enum {string} */
+// tslint:disable:object-literal-sort-keys no-bitwise
```

Comment:
does this disable the entire file? Do we need this?

Reply:
Removed file-level `tslint:disable` comments in favor of block- and line-level comments.